Dockerfile

```diff
@@ -1,9 +1,29 @@
 FROM continuumio/miniconda3:4.12.0
 
+ENV HOME="/root"
+WORKDIR ${HOME}
+
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-    libyaml-cpp-dev=0.6.3-9 libyaml-dev=0.2.2-1 default-libmysqlclient-dev=1.0.7 gcc=4:10.2.1-1 && \
-    apt-get clean && rm -rf /var/lib/apt/lists/*
+        libyaml-cpp-dev \
+        libyaml-dev \
+        git \
+        build-essential \
+        libffi-dev \
+        liblzma-dev \
+        libncurses-dev \
+        libreadline6-dev \
+        libsqlite3-dev \
+        libbz2-dev \
+        default-libmysqlclient-dev \
+        curl \
+        && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* && \
+    git clone --depth=1 https://github.com/pyenv/pyenv.git .pyenv
```

Comment:
we could do auto-installer https://github.com/pyenv/pyenv-installer#install
btw, do we need pyenv there?

Reply:
pyenv is used now by mlflow in order to get appropriate python version, and inference does not work without it

Reply:
I tried that installer at first, but it did not work correctly in docker somehow, so I did it the other way and managed to get it running. You may try to make it work with installer, though

Reply:
there was something regarding piping to bash and errors from apt install being piped there as well, which made debugging a bit tricky since it hides command output. With correct piping, it is possible to use the installer, or simpler yet - save the script to a file and execute it afterward.

Reply:
got you, thanks for the detailed explanation.
if it works, let's leave it as is :) 
if the PR is ready and works - please merge